constants/companies.ts

```diff
@@ -0,0 +1,202 @@
+  },
+  {
+    name: 'Inflexion Games',
+    to: 'https://www.inflexion.io/work-with-us',
+    description: 'Inflexion is a game development company focused on creating immersive and engaging gaming experiences. Developers here will work on innovative game mechanics, storytelling, and cutting-edge graphics technologies.',
+  },
+  {
+    name: 'Apple',
+    to: 'https://www.apple.com/careers/ca/',
+    description: 'Apple is a global leader in consumer electronics, offering products like the iPhone, Mac, and various software platforms. Developers at Apple work on iOS/macOS app development, hardware-software integration, and large-scale systems like iCloud.',
+  },
+  {
+    name: 'DigitalOcean',
+    to: 'https://www.digitalocean.com/careers',
+    description: 'DigitalOcean provides cloud infrastructure services targeted at small-to-medium-sized businesses and developers. Developers here will work on cloud infrastructure, scalable systems, and tools that make cloud computing more accessible to startups and individual developers.',
+  },
+  {
+    name: 'ATB Financial',
+    to: 'https://careers.atb.com/careers',
+    description: 'ATB Financial is a regional bank in Alberta, Canada, providing a range of financial services. Developers at ATB work on digital banking platforms, APIs, and customer-facing applications aimed at transforming the banking experience with tech-driven solutions.',
+  },
+  {
+    name: 'Invidi',
+    to: 'https://www.invidi.com/contact-us/careers/',
+    description: 'Invidi offers addressable advertising solutions, delivering targeted ads across digital and broadcast platforms. Developers will work on sophisticated adtech systems, integrating real-time data, analytics, and broadcast technologies.',
+  },
+  {
+    name: 'Rithmik',
+    to: 'https://rithmik.com/work-with-us/',
+    description: 'Rithmik develops AI-powered predictive maintenance solutions for industrial equipment. Developers here will be working on machine learning models and IoT solutions, aiming to prevent equipment failures and optimize industrial processes.',
+  },
+  {
+    name: 'Computronix',
+    to: 'https://www.computronix.com/',
+    description: 'Computronix offers enterprise software solutions for government organizations, focusing on digital transformation and workflow automation. Developers here will work on large-scale systems, including cloud solutions, workflow automation, and case management platforms.',
+  },
+  {
+    name: 'Paper Leaf',
+    to: 'https://paper-leaf.com/careers/',
+    description: 'Paper Leaf is a web development and design agency, creating custom web applications for businesses. Developers at Paper Leaf work on frontend and backend solutions, building responsive and user-friendly websites and apps for a variety of industries.',
+  },
```

Comment:
One I added late last week is ualberta too, if you want to add it that would be great. Otherwise I can create another issue to add it. 

https://apps.ualberta.ca/careers/list/category/50